pkg/controller/podautoscaler/horizontal.go

```diff
@@ -211,10 +213,15 @@ func (a *HorizontalController) processNextWorkItem() bool {
 	if err != nil {
 		utilruntime.HandleError(err)
 	}
-	// Add request processing HPA after resync interval just in case last resync didn't insert
-	// request into the queue. Request is not inserted into queue by resync if previous one wasn't processed yet.
-	// This happens quite often because requests from previous resync are removed from the queue at the same moment
-	// as next resync inserts new requests.
+	// Add request processing HPA to be executed after resyncPeriod. Requests are always added to
```

Comment:
"Add request processing HPA to be executed after resyncPeriod" does not parse.

Reply:
Is it better?